test/failure_app_test.rb

```diff
@@ -294,5 +294,22 @@ def call_failure(env_params={})
       assert @response.third.body.include?('<h2>Log in</h2>')
       assert @response.third.body.include?('Your account is not activated yet.')
     end
+
+    if Rails.application.config.respond_to?(:relative_url_root)
```

Comment:
If there are other tests relying on this pattern you have answered my question, it is for Rails compat. We can probably remove it in another PR if the build passes after doing so. :)